lib/ssl/doc/src/ssl.xml

```diff
@@ -299,6 +303,15 @@
       and to restrict their usage when using a cipher suite supporting them.
       </p></item>
 
+      <tag><c>{supported_groups, [group()]}</c></tag>
+      <item><p> Used only in TLS 1.3 and obsoletes the eccs option. It indicates the named
+      groups which the client supports for key exchange, ordered from most preferred to least
+      preferred. In case of a server it is used to update the client's view of its preferences.</p>
+      <warning><p>Note that in versions of TLS prior to TLS 1.3, this option only contained
```

Comment:
In the server case it is the groups supported by the server. I am not sure we want to explain how the server may send an extension to the client to hopefully update its precedence for next time. 

Maybe we should say: It indicates the named groups supports for key exchange. The server will select a common
group from the clients list which should be ordered  from most preferred to the least preferred.  

Or if we deem it important to the end user, that is handling of server extension should be handled by the user of ssl
application... maybe we shall split the doc of supported groups to client side and server side as many of the other options in ssl.

Reply:
The RFC says the client may use the supported_groups to update its key_share extension for future connections. But that doesn't appear to be possible with the current interface. So maybe this should be mentioned in a future commit only.

Reply:
Yes,  TLS-1.3 support is far from finished. So it could be a good idea to hold back the documentation commits that do not affect TLS-1.2,  even if TLS-1.3  its planned for OTP 22. 
 Although we need to continuously merge TLS-1.3 functionality, but it is not supported until its documented and we should not promise too much. But we do not want to save all documentation work for last, and we do appreciate comments on if the documentation makes sense to our users even if it is not fully implemented yet.

Reply:
Will fix.